core/graph/connectors/socket_server.py

```
@@ -0,0 +1,370 @@
+            loop = asyncio.get_event_loop()
+            result = await loop.run_in_executor(
+                None,
+                analyze_interdisciplinary_connections,
+                min_year_gap,
+                min_similarity
+            )
+            
+            return {
+                'status': 'analysis_complete',
+                'result': result
+            }
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)